src/IECore/StreamIndexedIO.cpp

```diff
@@ -1137,9 +1327,71 @@ void StreamIndexedIO::Node::removeChild( const IndexedIO::EntryID &childName, bo
 //
 ///////////////////////////////////////////////
 
-StreamIndexedIO::Index::Index( StreamIndexedIO::StreamFilePtr stream ) : m_root(nullptr), m_version(g_currentVersion), m_hasChanged(false), m_offset(0), m_next(0), m_stream(stream)
+StreamIndexedIO::Index::Index( StreamIndexedIO::StreamFilePtr stream, const CompoundData *options )
+	: m_root( nullptr ),
+	m_version( g_currentVersion ),
+	m_hasChanged( false ),
+	m_offset( 0 ),
+	m_next( 0 ),
+	m_stream( stream ), m_compressionLevel( 9 ),
+	m_compressionThreadCount(1),
+	m_decompressionThreadCount(1), m_compressor( "lz4" )
```

Comment:
Does this mean that compression is now on by default?

Reply:
I was hoping to enable compression by default and disable it if it caused problems.